code/mission/missionparse.cpp

```diff
@@ -3967,7 +3975,8 @@ int parse_wing_create_ships( wing *wingp, int num_to_create, int force, int spec
 				wingp->flags |= WF_WING_GONE;
 
 				// if the current wave is zero, it never existed
-				wingp->flags |= WF_NEVER_EXISTED;
+				if (wingp->current_wave == 0)
+					wingp->flags |= WF_NEVER_EXISTED;
```

Comment:
I think the use of WF_NEVER_EXISTED should be audited to see if its name matches its function, as I have a sneaking suspicion that it doesn't.